site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -7,7 +7,7 @@ The optional Id of an existing Tenant to make a copy of. If present, the [field]
 [field]#tenant.connectorPolicies# [type]#[Array]# [optional]#Optional# [since]#Available since 1.18.0#::
 A list of Connector policies. Users will be authenticated against Connectors in order. Each Connector can be included in this list at most once and must exist.
 
-[field]#tenant.connectorPolicies``[x]``.connectorId# [type]#[UUID]# [required]#Required# [since]#Available since 1.18.0#::
+[field]#tenant.connectorPolicies``[x]``.connectorId# [type]#[UUID]# [optional]#Optional# [default]#defaults to the FusionAuth connector Id# [since]#Available since 1.18.0#::
```

Comment:
The FusionAuth connector Id is fixed and is equal to `e3306678-a53a-4964-9040-1c96f36dda72` - should we just state this instead?